c7n/utils.py

```diff
@@ -343,3 +344,11 @@ def _f(*args, **kw):
             raise
     functools.update_wrapper(_f, f)
     return _f
+
+
+def timestamp_from_string(date_text):
```

Comment:
given specialized usage on this i think just move to log support mod and maybe do the * 1000 there as well.